ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -21,17 +21,19 @@ set +e
 # it wastes flash
 #
 set AUX_MODE pwm
-set BOARD_RC /etc/init.d/rc.board
+set BOARD_RC_DEFAULTS /etc/init.d/rc.board_defaults
+set BOARD_RC_EXTRAS /etc/init.d/rc.board_extras
+set BOARD_RC_SENSORS /etc/init.d/rc.board_sensors
```

Comment:
Maybe rename to `rc.board_early` and `rc.board_late`? Or `pre` and `post`?

Reply:
I thought about something similar and then even considered run levels (including shutdown), but ultimately settled on functionality to help with startup positioning, maintaining consistency between boards, and some level of self documentation.

 - single file for parameters
 - single file for sensors
 - extras originally had a different name, but then needed to go before logging, and also seemed like going before the sdcard addons script (aka FEXTRAS) might be prudent.

Reply:
I don't feel strong and the current naming makes sense, but I think there will be other reasons to put something other than params in the params/defaults file. A pre and post would then make it more clear when it's executed.